### ISSUE TITLE
Source not working with 2021.3.x version

### ISSUE THREAD

**Issue by benrajsdKiksARVR** [Unity-Technologies/FontainebleauDemo#21]
**Describe the bug**
Cloned the repo and opened the project in Unity 2021.3.6f1 version, its not working, there is a error in console about a issue on HDRP package, but in package manager there is no error or warning. Game-play view is white, nothing happened with post processing/fog disable.

**Editor or Runtime**
Editor and Runtime

**To Reproduce**
Steps to reproduce the behaviour:
1. Clone the project and open in 2021.3.6f1

**Expected behavior**
Should work as expected

**Platform**
 - Device: PC
 - OS: Windows
 - Version 11

**Comment by sebastienlagarde** [comment]
Hi, this project only works with 2019.3 and haven't been upgraded to more recent version

**Comment by benrajsdKiksARVR** [comment]
Okay got it, any plan for upgrade to latest Unity versions.


--
Regards,
Benraj SD
Product Architect | KiXR Technologies Pvt. Ltd.

3rd floor, #183, Gayathri Tech Park | Road 1B, EPIP Zone, White Field |
Bangalore-560066
Email : ***@***.*** | Mobile : +91 9633102979
Web :http://www.kiksarvr.com

**Comment by sebastienlagarde** [comment]
No, the support for this scene have been stopped and no futher change will be done.

**Comment by benrajsdKiksARVR** [comment]
Okay got it


--
Regards,
Benraj SD
Product Architect | KiXR Technologies Pvt. Ltd.

3rd floor, #183, Gayathri Tech Park | Road 1B, EPIP Zone, White Field |
Bangalore-560066
Email : ***@***.*** | Mobile : +91 9633102979
Web :http://www.kiksarvr.com
